### ISSUE TITLE
Animation is not properly updated after a category is selected

### ISSUE THREAD

**Issue by ramiroaznar** [CartoDB/cartoframes#1359]
There is an unexpected animation behavior related to the interaction of filtering with widgets in cartoframes. After selecting one category in one widget, the time series widget histogram is redrawn but the animation does not adjust to the new state.

**STR**

1. Go to https://carto.com/blog/real-estate-data-tinsa-digital/
2. Check cartoframes map
3. Check the animation
4. Select a category
5. Check again the animation

**Current Result**

The yellow geometries are popping up ca. 1971.

![1](https://user-images.githubusercontent.com/5215798/70804936-07301100-1db8-11ea-918a-850d5ce1a7b1.png)

After `1971-1980` category is selected. The histogram of the time series widget is redrawn. But the animation is still behaving as nothing has been filtered. The geometries appear not in 1971 based on the current state of the widget, but around 1977. If you check it is the same point that the 1971 mark was set in the previous state.

![2](https://user-images.githubusercontent.com/5215798/70804976-22028580-1db8-11ea-8264-b62006df788f.png)

**Expected Result**

The geometries should start popping up at the very beggining of the animation (bar) after the selection.

**Environment**

Ubuntu 18
Chrome v79

**Comment by ramiroaznar** [comment]
Hey @elenatorro, checking on the blogpost map it does not seem the issue is fixed.

**Comment by elenatorro** [comment]
This issue was closed automatically after the merge, sorry. The solution will be included in the next Airship release.

**Comment by elenatorro** [comment]
Opened issue for RT in the blog repo https://github.com/CartoDB/cartoframes/issues/1359

**Comment by ramiroaznar** [comment]
I dont see any improvements in the blog post (the geometries have changed though). Maybe the cartoframe anination was done by a previous version...

![cartoframes](https://user-images.githubusercontent.com/5215798/73002882-fdf98180-3e04-11ea-8ea4-d2ac3d573661.gif)

**Comment by elenatorro** [comment]
Hi @ramiroaznar :) As you said, the animation has changed. This specific issue has been solved, but it's true that we need to be able to configure the animation through CARTOframes. In this blog post example, the problem is that the animation doesn't _hold_.

For this purpose, we've opened [this issue](https://github.com/CartoDB/airship/issues/675), in order to use an animation helper in the styles.

Thank you very much! :)
